deployment/ansible/roles/dcld/tasks/init.yml

```diff
@@ -0,0 +1,34 @@
+---
+# Copyright 2022 DSR Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+- name: create DCL home folder
+  file:
+    mode: '0755'
+    state: directory
+    path: "{{ dcld.home }}"
+
+- name: check that genesis file exists
+  stat:
+    path: "{{ dcld.home }}/config/genesis.json"
+  register: genesis_stat
+
+- name: print genesis_stat value
+  debug:
+    var: genesis_stat
+
+- name: init node configuration files
+  become: true
+  shell: "{{ dcld.path }} init {{ ansible_hostname }} --chain-id {{ dcld.config.chain_id }} --overwrite --home {{ dcld.home }}"
```

Comment:
I believe `--overwrite` is not needed here since we rely on missed genesis.json here.
`--home` makes sense to drop as well (please check related comment for `dcld.service.j2`)